lib/io/can.hpp

```diff
@@ -3,6 +3,8 @@
 #include <core/types.hpp>
 
 namespace hyped::io {
+// every frame sent to the controller contains 8 bytes
+static constexpr std::uint8_t kControllerCanFrameLength = 8;
```

Comment:
Idk much about can  but is this something that will only really be used here or will sensors need to use this value too (i.e. should it live here or should it live somewhere more accessible?)

Reply:
I don't know what length frames the BMS uses. 8 bytes is the maximum length for a (non-extended) CAN frame, but they might be using shorter frames. With #21 the struct for a can frame will be in can.hpp so I think this is the right place for it.

@SnickeyX will all your BMS CAN frames be 8 bytes long?

Reply:
I just realised that this is in `io`... surely that is THE most accessible place to have it?

Reply:
they seem to be 8 bytes for BMS, but ideally we want these to variable because more sensors are going to be on CAN apparently and I don't yet know what their frame sizes are (waiting on a confirmation of the sensor choice atm)

Reply:
there is also already a `CAN_MAX_DLEN` macro defined in the linux header which is set to 8. i think this is a const for mtc only and other cases should define their own sizes in their own places